### ISSUE TITLE
r-universe vignettes failed to build

### ISSUE THREAD

**Issue by PietrH** [inbo/etn#326]
https://github.com/r-universe/inbo/actions/runs/11086040208

![image](https://github.com/user-attachments/assets/ac93c065-d736-4a66-9db7-76d7ec2f57dd)

As reported by Thierry

Probably purely due to the vignettes being dependent on a database connection/credentials for the API. 

@peterdesmet Could we pre-render the vignettes for r-universe?

**Comment by peterdesmet** [comment]
Right, we could use the frictionless approach and pre-render our vignettes, see https://github.com/frictionlessdata/frictionless-r/blob/main/vignettes/precompile.R

**Comment by PietrH** [comment]
2 options, either pre-render our vignettes, like frictionless-r, see: https://ropensci.org/blog/2019/12/08/precompute-vignettes/

Or use vcr cassettes, and use a stored HTTP response: 
https://docs.ropensci.org/vcr/reference/setup_knitr.html
